crd-generator/src/main/java/io/strimzi/crdgenerator/CrdGenerator.java

```diff
@@ -647,6 +645,10 @@ private void checkClass(Class<?> crdClass) {
             }
         }
 
+        if (crdClass.getName().startsWith("io.strimzi.")) {
+            checkJsonInclude(crdClass);
+            checkJsonPropertyOrder(crdClass);
+        }
```

Comment:
I recall having it under the existing `io.strimzi.api` check broke some tests, but let me verify

Reply:
That seems weird. I would not have expected it to make any difference. Which test fails?

Reply:
I encountered an error when trying to remove the `if` condition:

```
CrdGenerator: error: class io.fabric8.kubernetes.api.model.Quantity is missing @JsonInclude
CrdGenerator: error: class io.fabric8.kubernetes.api.model.Quantity is missing @JsonPropertyOrder
```
During the build process of `api 0.43.0-SNAPSHOT` using the command `make all`

Reply:
@KastTrifork Right. But if you move the two methods instead two lines down into the condition `if (crdClass.getName().startsWith("io.strimzi.api.")) {` it should not break anything related to `io.fabric8`, or?

Reply:
The if-condition below it is IMHO there for exactly the same purpose -> to avoid doing the check (in this case `checkInherits`) for the Fabric8 classes.

Reply:
Yes, you are right, Scholz. I misunderstood what you wrote. I will make the change now :-)